docs/website/data/integrations.yaml

```diff
@@ -1012,6 +1012,29 @@ integrations:
     blogs:
       - https://github.com/EliuX/flask-opa/tree/master/examples
 
+  permit:
```

Comment:
You will need to create an entry for `permit` in the [software](https://github.com/open-policy-agent/opa/blob/main/docs/website/data/integrations.yaml#L1306) object.